src/ucp/api/ucp.h

```diff
@@ -2100,6 +2123,33 @@ ucs_status_t ucp_context_query(ucp_context_h context_p,
                                ucp_context_attr_t *attr);
 
 
+/**
+ * @ingroup UCP_MEM
+ * @brief Compare two remote keys
+ *
+ * This routine returns equal if all the transports can compare equal and
+ * configurations are equal.
+ *
+ * The comparison result is < 0 if rkey1 is lower than rkey2, 0 if they
+ * are equal or > 0 if rkey1 is greater than rkey2. The remote key greater-than
+ * semantic is arbitrary and implementation dependent, nevertheless it's
```

Comment:
`greater-than semantics` ?

Reply:
`arbitrary and implementation dependent` ? arbitrary == random .  I don't think the semantics can be random and consistent in the same time.

Reply:
based on comments, replaced by `it can be used for sorting`.